apps/researcher/src/app/[locale]/page.tsx

```diff
@@ -36,17 +37,43 @@ import {ElementType} from 'react';
 // Revalidate the page every n seconds
 export const revalidate = 60;
 
-interface FacetProps {
+interface FilterSetting {
   name: keyof SearchResult['filters'];
   searchParamType: SearchParamType;
+}
+
+const filterSettings: ReadonlyArray<FilterSetting> = [
+  {name: 'owners', searchParamType: 'array'},
+  {name: 'types', searchParamType: 'array'},
+  {name: 'subjects', searchParamType: 'array'},
+  {name: 'publishers', searchParamType: 'array'},
+  {name: 'dateCreatedStart', searchParamType: 'number'},
+  {name: 'dateCreatedEnd', searchParamType: 'number'},
+];
+
+interface Facet {
+  name: string;
   Component: ElementType;
 }
 
-const facets: ReadonlyArray<FacetProps> = [
-  {name: 'owners', searchParamType: 'array', Component: FilterSet},
-  {name: 'types', searchParamType: 'array', Component: FilterSet},
-  {name: 'subjects', searchParamType: 'array', Component: FilterSet},
-  {name: 'publishers', searchParamType: 'array', Component: FilterSet},
+interface DateRangeFacet {
```

Comment:
Is it possible to extend the `Facet` interface here? Then `DateRangeFacet` can make use of `name` and `Component`?

Reply:
Yes, I agree that that is better